controlplane/rosa/api/v1beta2/rosacontrolplane_webhook.go

```diff
@@ -56,6 +60,14 @@ func (r *ROSAControlPlane) ValidateCreate() (warnings admission.Warnings, err er
 	)
 }
 
+func (r *ROSAControlPlane) validateClusterRegistryConfig() *field.Error {
+	if len(r.Spec.ClusterRegistryConfig.RegistrySources.AllowedRegistries) > 0 && len(r.Spec.ClusterRegistryConfig.RegistrySources.BlockedRegistries) > 0 {
```

Comment:
need to check if `ClusterRegistryConfig` and `RegistrySources` is nil first.

Reply:
done